ios/EnrichedTextInputView.mm

```diff
@@ -837,6 +842,12 @@ - (void)setValue:(NSString *)value {
   [self anyTextMayHaveBeenModified];
 }
 
+- (void)setCustomSelection:(NSInteger)start end:(NSInteger)end {
+  NSInteger amountOfZeroWidthSpaces = [ZeroWidthSpaceUtils getAmountOfZeroWidthSpacesInRange:NSRange(start, end - start) text:textView.textStorage.string];
+    
+  textView.selectedRange = NSRange(start, end + amountOfZeroWidthSpaces - start);
```

Comment:
We rather use `NSMakeRange`. Zero width spaces are countable so it should be `number`, not `amount`.
```suggestion
  NSInteger numberOfZeroWidthSpaces = [ZeroWidthSpaceUtils getNumberOfZeroWidthSpacesInRange:NSMakeRange(start, end - start) text:textView.textStorage.string];
    
  textView.selectedRange = NSMakeRange(start, end + numberOfZeroWidthSpaces - start);
```